### PR TITLE
Final Contribution Blog Post

### PR DESCRIPTION
**Pull request by h3allen**

### CODE REVIEW

**Comment by MoralCode** [comment]
Looks good overall! I notice you didn't include any screenshots or images in your blog post though - would you like me to merge this post without them? 2 of the 10 total points in the rubric for the blog posts are for "appropriate use of images"

**Comment by h3allen** [comment]
Ack! I thought I had. Oops! I'll fix that right now.

File: _posts/2024-05-04-vertexbuffer-docs-fix.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+The issue has also been up since 2020, so either no one has bothered to fix it, or no one’s made an adequate change to the documentation that actually clarifies the SetData method. 
+The issue appeared to be something I could solve with a little bit of effort and would mostly rely on me figuring out how MonoGame's documentation system works. I have experience with handling (and screwing up) vertex buffers, so it was something well within my wheelhouse. The biggest task was actually finding what the docs were describing and working through what the code actually did.
+
+The same two maintainers I saw on my bug fix have also interacted with this issue organizationally. It’s gone from being a milestone for 3.8.1 to just an item on a To Do list for documentation. From the looks of it, despite MonoGame listing on their website that they swap maintainers monthly, these two will continue to be managing the repos for the time being.
+
+I commented on the issue thread and started the process of improving the lack of documentation.
+
+![Issue by georg-eckert-zeiss describing lack of detail in examples for Set Data in Vertex Buffer for MonoGame](../assets/2024-05-04-Allen/Issue#7396.png)
```
Comment (MoralCode) [inline _posts/2024-05-04-vertexbuffer-docs-fix.md:36]: # is not typically allowed in URLs and is probably why this image isnt working

Comment (MoralCode) [reply _posts/2024-05-04-vertexbuffer-docs-fix.md:36]: @h3allen not sure if you got a notification for the above so i figured a ping may be useful

Comment (h3allen) [reply _posts/2024-05-04-vertexbuffer-docs-fix.md:36]: They showed up for me the second time (I think?)

Comment (h3allen) [reply _posts/2024-05-04-vertexbuffer-docs-fix.md:36]: Haha nevermind

**Pull request merged**